### DISCUSSION TITLE
Users are stored in database, groups are not

### DISCUSSION THREAD

**Discussion by stefan-1st** [statamic/cms#6582] [Troubleshooting] [answered]
Hello everyone,

I have a problem with the "store users in the database" approach. I followed the documentation and everything seems to work. But checking the database, the users, role assignment, and group assignment tables are populated. There is no table for the groups and roles itself. Is it an issue with my configuration or is it supposed to work like that?

It is important for us to store those kind of information in the database.

I have php 8.0.19 running, MySQL 8, Statamic 3.3.27, Laravel 9.24.0

The search resulted in the discussion linked below. Configuring the group and the roles table would lead to access to non existing tables since they are not created in any migration.

https://github.com/statamic/cms/discussions/6392

**Comment by jasonvarga** [comment, marked as answer]
See https://github.com/statamic/cms/pull/5686

**Reply by stefan-1st** [reply]
Ah, thanks @jasonvarga . I didn't recognize the PR isn't accepted yet. Thanks for the clarification.
